### ISSUE TITLE
Accessibility: Datepicker remains in tab-loop

### ISSUE THREAD

**Issue by RGarbersWPS** [vmware/clarity#5189]
## Describe the bug

It is currently not possible to tab through a form where at least one datepicker is in. You will not be able to go further than the datepicker as you will be stuck in a tab-loop.

## How to reproduce

1.  Go to a datepicker
2.  Try to tab through it
3.  Get stuck

You could try it on the Clarity Datepicker Documentation page.

## Expected behavior

You should be able to tab over a datepicker or at least leave the loop with another key like Enter, Esc, ...

## Versions

Clarity 4.0.4

## Additional notes

It worked before I upgraded from 3.x to 4.x

**Comment by coryrylan** [comment]
Thanks for filing this, I was able to reproduce the issue.

**Comment by amellnik** [comment]
Related: #5136

**Comment by amellnik** [comment]
Like #5136 this now appears fixed (unsure what PR actually fixed it).  Triggers are no longer opening automatically which allows users to tab past these.  I think this can be closed.

**Comment by Jinnie** [comment]
Thanks for checking and reporting this - I am also unable to reproduce it.
